dockers/docker-base-stretch/sources.list

```diff
@@ -1,8 +1,8 @@
 ## Debian mirror on Microsoft Azure
 ## Ref: http://debian-archive.trafficmanager.net/
 
-deb http://debian-archive.trafficmanager.net/debian/ stretch main contrib non-free
-deb-src http://debian-archive.trafficmanager.net/debian/ stretch main contrib non-free
-deb http://debian-archive.trafficmanager.net/debian-security/ stretch/updates main contrib non-free
-deb-src http://debian-archive.trafficmanager.net/debian-security/ stretch/updates main contrib non-free
-deb http://debian-archive.trafficmanager.net/debian/ stretch-backports main contrib non-free
+deb [arch=amd64,i386] http://debian-archive.trafficmanager.net/debian/ stretch main contrib non-free
```

Comment:
we do not need i386, only amd64 is enough. I suspect this is reason the 13M increase of image size.

Reply:
Done